sdk/monitor/Azure.Monitor.OpenTelemetry.LiveMetrics/src/Internals/Diagnostics/LiveMetricsExporterEventSource.cs

```diff
@@ -0,0 +1,69 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+using Azure.Monitor.OpenTelemetry.Exporter.Internals;
+
+namespace Azure.Monitor.OpenTelemetry.LiveMetrics.Internals.Diagnostics
+{
+    /// <summary>
+    /// EventSource for the AzureMonitorExporter.
+    /// EventSource Guid at Runtime: 72f5588f-fa1c-502e-0627-e13dd2bd67c9.
```

Comment:
Is this a GUID for OpenTelemetry-AzureMonitor-LiveMetrics?

Reply:
yes! i debugged to capture the guid